.github/workflows/on-push.yml

```diff
@@ -317,6 +317,14 @@ jobs:
           cd content
           demisto-sdk pre-commit --validate --show-diff-on-failure --verbose -i Packs/HelloWorld -i Packs/QRadar/Integrations/QRadar_v3 --mode=nightly
 
+      - name: run pre-commit with default pre-commit config shipped with the SDK
+        timeout-minutes: 60
+        run: |
+          source $(poetry env info --path)/bin/activate
+          cd content
+          rm -rf .pre-commit-config_template.yaml
+          demisto-sdk pre-commit --validate --show-diff-on-failure --verbose --mode=ci
```

Comment:
can we capture output, make sure some custom named hook gets run?

Reply:
which hooks for example?

Reply:
pytest or anything, just give a name so it doesnt run exit 0 silently without doing anything

Reply:
can you explain what exactly do you want here?